server/balancer_worker.go

```diff
@@ -169,10 +174,12 @@ func (bw *balancerWorker) doBalance() error {
 		}
 		if balanceOperator == nil {
 			stats.Increment("balance.select.none")
-			return nil
+			continue
```

Comment:
I only want to build balancer operations more quickly,  maybe we can discuss it.
@siddontang  what is your opinion?